docs/Flow.md

```diff
@@ -20,4 +20,44 @@ Need to redesign this approach the way to handle every possible change of object
 - Creation of bookmark
 - ...
 
-As now backlinks are gotten from the object store, all these changes need to be done after store redesign, so mechanism of getting links of object and saving it both to store and to backlinks relation has same enter point.
\ No newline at end of file
+As now backlinks are gotten from the object store, all these changes need to be done after store redesign, so mechanism of getting links of object and saving it both to store and to backlinks relation has same enter point.
+
+## Internal Flags
```

Comment:
Maybe make it as comments in code? This separate documentation will decay over time, much faster than commented code